contracts/controller/PowerEnabled.sol

```diff
@@ -93,7 +93,7 @@ contract PowerEnabled is MarketEnabled {
     uint256 outstandingPow = outstandingPower();
     require(outstandingPow.add(amountPow) <= maxPower);
     uint256 powBal = powerBalanceOf(_from).add(amountPow);
-    require(powBal >= authorizedPow.div(MIN_SHARE_OF_POWER));
+    //require(powBal >= authorizedPow.div(MIN_SHARE_OF_POWER));
```

Comment:
We could just have a `require(_amountBabz >=  minPowerUpSizeBabz())` at the top of the function.